### ISSUE TITLE
[Anon] Bug report: Imperium - Astra Militarum.catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k-9e#9333]
**File:** Imperium - Astra Militarum.catz

**BattleScribe version:** 2.03.21

**Platform:** Windows

**Dropbox:** No

**Description:** The Earthshaker Battery [Legends] (Which is supposed to be the Earthshaker Platform) is filling in with the Earthshaker Carriage Battery.

It also maintains the post cost of the Battery (125) when the Carriage is 120, even though it's filling in the Earthshaker Carriage Battery for both of the choices.

Basically, the Earthshaker Battery [Legends] is supposed to come with an Earthshaker Platform, and not an Earthshaker Carriage Battery when selected.  Both can be found in the Heavy Support Tab for Astra Militarum.

Forwarding a file with both items,

Thank you in advance.

**Comment by Technoblazed** [comment]
Your assumption is incorrect.

The Earthshaker Battery [LEGENDS] consists of numerous Earthshaker Carriage units, of which each is 125 points.
https://www.warhammer-community.com/wp-content/uploads/2020/11/1qpk5gayVm5VmR4f.pdf

The Earthshaker Carriage Battery, which has separate Earthshaker Carriage units, are 120 points each.
